src/components/ebay-fake-menu-button/component.js

```diff
@@ -41,11 +41,17 @@ export default {
         this.emitComponentEvent({ eventType: "mousedown", el, originalEvent });
     },
     emitComponentEvent({ eventType, el, originalEvent, index }) {
-        const eventObj = {
+        let eventObj = {
             el,
             originalEvent,
             index,
         };
+        eventObj = Object.entries(eventObj)
```

Comment:
If you'd like you can use `Object.fromEntries` here instead of `reduce`. I think it's a little cleaner but it is personal preference.

Reply:
I like it. Looks cleaner. Rolled out the change.